generic-file-system/api/src/test/java/org/pentaho/platform/api/genericfile/GetTreeOptionsTest.java

```diff
@@ -435,20 +406,19 @@ void testEqualsItself() {
     @Test
     void testDoesNotEqualNull() {
       GetTreeOptions options = new GetTreeOptions();
-      assertFalse( options.equals( null ) );
+      assertNotEquals( null, options );
```

Comment:
```suggestion
      assertNotNull( options );
```

Reply:
Actually, in this case, I should not have replaced the original statement, as I did with others. Here, I think it's important to keep the original form.

Reply:
Reverted in https://github.com/pentaho/pentaho-scheduler-plugin/pull/243/commits/75b3bd168c4084e78908e5ac4431c576bed56446.